spec/models/post_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  first_user = User.new(name: 'Tom',
+                        photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+
+  it 'comments_counter should not be less than 0' do
+    first_post = Post.new(author: first_user, title: 'Hello', text: 'This is my first post')
+    first_post.comments_counter = -3
+    expect(first_post).to_not be_valid
+  end
+
+  it 'comments_counter should be an integer' do
+    first_post = Post.new(author: first_user, title: 'Hello', text: 'This is my first post')
+    first_post.comments_counter = 'one'
+    expect(first_post).to_not be_valid
+  end
+
+  it 'likes_counter should be an integer' do
+    first_post = Post.new(author: first_user, title: 'Hello', text: 'This is my first post')
+    first_post.likes_counter = 'one'
+    expect(first_post).to_not be_valid
+  end
```

Comment:
- [ ] Kindly add test(s) for a post title validation.
- [ ] Kindly add tests for the model methods such as __five_most_recent_comments__.


![methods](https://github.com/WilhelmK109/BlogApp/assets/53124548/2a780b5a-d58e-4a26-9fb8-bf16a8d64fad)